src/Sulu/Bundle/ContentBundle/Command/WebspaceCopyCommand.php

```diff
@@ -0,0 +1,667 @@
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+
+/**
+ * Copies a given webspace with given locale to a destination webspace with a destination locale.
+ */
+class WebspaceCopyCommand extends ContainerAwareCommand
+{
+    /**
+     * @var OutputInterface
+     */
+    private $output;
+
+    /**
+     * @var DocumentManager
+     */
+    private $documentManager;
+
+    /**
+     * @var SessionManagerInterface
+     */
+    private $sessionManager;
+
+    /**
+     * @var DocumentInspector
+     */
+    private $documentInspector;
+
+    /**
+     * @var HtmlTagExtractor
+     */
+    private $htmlTagExtractor;
+
+    /**
+     * @var string
+     */
+    protected $webspaceKeySource;
+
+    /**
+     * @var string
+     */
+    protected $webspaceKeyDestination;
+
+    protected function configure()
+    {
+        $this->setName('sulu:webspaces:copy')
+            ->addArgument('source-webspace', InputArgument::REQUIRED)
+            ->addArgument('source-locale', InputArgument::REQUIRED)
+            ->addArgument('destination-webspace', InputArgument::REQUIRED)
+            ->addArgument('destination-locale', InputArgument::REQUIRED)
+            ->setDescription('Copy whole webspace');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $webspaceKeySource = $input->getArgument('source-webspace');
+        $localesSource = explode(',', $input->getArgument('source-locale'));
+        $webspaceKeyDestination = $input->getArgument('destination-webspace');
+        $localesDestination = explode(',', $input->getArgument('destination-locale'));
+
+        if (count($localesSource) !== count($localesDestination)) {
+            $output->writeln(
+                '<info>Processing aborted: </info>' .
+                '<comment>Provide correct source and destination locales</comment>'
+            );
+
+            return;
+        }
+
+        $output->writeln([
+            '<info>Webspace Copy</info>',
+            '<info>==============================</info>',
+            '',
+            '<info>Options</info>',
+            '------------------------------',
+            'Webspace: ' . $webspaceKeySource . ' => ' . $webspaceKeyDestination,
+        ]);
+
+        $localesPairs = [];
+        for ($i = 0; $i < count($localesSource); ++$i) {
+            $localesPairs[] = $localesSource[$i] . ' => ' . $localesDestination[$i];
+        }
+        $output->writeln([
+            'Locales: ' . implode(', ', $localesPairs),
+            '---------------',
+            '',
+        ]);
+
+        $helper = $this->getHelper('question');
+        $question = new ConfirmationQuestion('<question>Continue with this options?(y/n)</question> ', false);
+
+        if (!$helper->ask($input, $output, $question)) {
+            $output->writeln('<error>Abort!</error>');
```

Comment:
I've copied this from another command.. in my opinion you are right.. it's no error.. but an info that a user has aborted the command is legit..